### ISSUE TITLE
Embedding Projector Format

### ISSUE THREAD

**Issue by korakot** [Kyubyong/wordvectors#1]
Please help make a pre-trained model that can be used with Google's [Embedding Projector](http://projector.tensorflow.org/).
As I understand, it's 1 binary tensor file and 1 corresponding label file.
It's very cool if people can visualize and explore them instantly.
I did 2 example gists with [iris](https://gist.github.com/korakot/c480edd1fcf7e02c49ccddbf5ac43fb9) and [thai novel](https://gist.github.com/korakot/e176567e2b0a4faeed425a4dd7772718).
